pkg/kubelet/dockershim/helpers_unsupported.go

```diff
@@ -18,6 +18,40 @@ limitations under the License.
 
 package dockershim
 
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+	dockertypes "github.com/docker/engine-api/types"
+	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
+)
+
 func DefaultMemorySwap() int64 {
 	return -1
 }
+
+func (ds *dockerService) getSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, separator rune) ([]string, error) {
```

Comment:
I see. Since those functions are for 'adding additional features', suppose it would be better to do no-op but leave a warning message instead. And in this case on osx it would also behavior same as on windows. What do you think? Please see updated commit for detail.